aicspylibczi/CziFile.py

```diff
@@ -401,31 +475,17 @@ def read_image(self, **kwargs):
         not match the m_index that is being used in libCZI or displayed in Zeiss' Zen software.
         """
         plane_constraints = self.czilib.DimCoord()
-        [plane_constraints.set_dim(k, v) for (k, v) in kwargs.items() if k in CziFile.ZISRAW_DIMS]
+        [
+            plane_constraints.set_dim(k, v)
+            for (k, v) in kwargs.items()
+            if k in CziFile.ZISRAW_DIMS
+        ]
```

Comment:
I think I now see that the lines I was commenting on above are all the same python one-liner. I still stand by my statement that this "one-liner" should be moved to its own function haha.